config/features.php

```diff
@@ -2,33 +2,35 @@
 
 return [
     (object)[
-        'title' => 'Authentication and Single sign-on',
-        'description' => 'Let your users use it in a snap!',
-        'image' => '/themes/bimbala/images/authentication.png'
+        'title' => 'SEO friendly & fast',
+        'description' => 'Simple and effection pages, making them fast and search engine friendly.',
+        'image' => '/themes/bimbala/images/homepage/features/cloud.svg'
     ],
     (object)[
         'title' => 'User Roles',
         'description' => 'Grant your team permissions based on roles, so they can do their job as they should.',
-        'image' => '/themes/bimbala/images/roles.png'
+        'image' => '/themes/bimbala/images/homepage/features/userroles.svg'
     ], 
     (object)[
-        'title' => 'Notifications *',
-        'description' => 'Easy to subscribe notification system, which can provide the needed updates, when you need them.',
-        'image' => '/themes/bimbala/images/notifications.png'
+        'title' => 'Integrations',
+        'description' => 'With the help of our integration or Zapier, you can use your data as you wish.',
```

Comment:
```suggestion
        'description' => 'With the help of our integrations in Zapier, you can use your data as you wish.',
```

Reply:
It should have been: With the help of our integrations or Zapier, you can use your data as you wish.

The idea is that you can use Zapier for some stuff or you can use our integrations for some stuff.